CHANGELOG.md

```diff
@@ -2,6 +2,25 @@
 
 Nothing Yet!
 
+# Version 0.2.0-prerelease.1 (2023-08-17)
```

Comment:
If you remove this heading (and the "Nothing Yet!"), cargo-dist will detect that the "Unreleased" heading applies to the prerelease, so we don't need to churn the heading all the time.